mypy/partially_defined.py

```diff
@@ -171,7 +173,7 @@ class ScopeType(Enum):
     Global = 1
     Class = 2
     Func = 3
-    Generator = 3
+    Generator = 4
```

Comment:
This didn't matter until we needed to handle generators differently. Generators actually do inherit the scope!